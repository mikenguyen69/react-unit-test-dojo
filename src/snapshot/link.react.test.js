import React from 'react'
import Link from './link.react'
import Renderer from 'react-test-renderer'

it('renders correctly', () => {
    const tree = Renderer
    .create(<Link page="http://www.facebook.com">Facebook</Link>)
    .toJSON();
    expect(tree).toMatchSnapshot();
})

it('renders another correctly', () => {
    const newTree = Renderer
    .create(<Link page="https://dotnetsharing.wordpress.com">My personal blog</Link>)
    .toJSON()
    expect(newTree).toMatchSnapshot()
}) 