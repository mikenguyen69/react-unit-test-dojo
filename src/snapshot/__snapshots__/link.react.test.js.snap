// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders another correctly 1`] = `
<a
  className="normal"
  href="https://dotnetsharing.wordpress.com"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  My personal blog
</a>
`;

exports[`renders correctly 1`] = `
<a
  className="normal"
  href="http://www.facebook.com"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  Facebook
</a>
`;
